include/common/s1ap_structs.h

```diff
@@ -406,6 +426,11 @@ typedef union nas_pdu_elements {
 	esm_msg_container esm_msg;
 	guti mi_guti;
 	bool esm_info_tx_required;
+}nas_pdu_elements_union;
+
+typedef struct nas_pdu_elements {
+   NasIeType_e msgType;
```

Comment:
Avoid Hungarian notation. Please follow coding style as in the file. Mostly linux kernel coding style is followed in openmme.

Reply:
Request to please mention which structure you want the change for.